### PR TITLE
feat: Addition of stripe test client id

### LINKED ISSUE #3391: Admin Area: Addition of Test Client Id in stripe
Addition of field for `Stripe_Test_Client_ID` as referred in https://github.com/fossasia/open-event-server/issues/6345

### PR DESCRIPTION
**Pull request by kushthedude**
Fixes #3391 
Blocked By https://github.com/fossasia/open-event-server/pull/6358

### CODE REVIEW

File: app/templates/components/forms/admin/settings/payment-gateway-form.hbs [hunk 1]
```diff
@@ -33,15 +27,24 @@
...
+        <label>
+          {{t 'Client Test ID'}}
+          {{ui-popup tagName="i" class="info icon" content="This client id will be used for testing mode"}}
+        </label>
+        {{input type='text' name='stripe_test_client_id' value=settings.stripeTestClientId}}
+      </div>
       <div class="field">
         <label>
           {{t 'Secret Test Key'}}
+          {{ui-popup tagName="i" class="info icon" content="This secret key will be used for testing mode"}}
```
Comment (CosmicCoder96) [inline app/templates/components/forms/admin/settings/payment-gateway-form.hbs]: translation helpers

File: app/templates/components/forms/admin/settings/payment-gateway-form.hbs [hunk 2]
```diff
@@ -33,15 +27,24 @@
...
         </label>
         {{input type='text' name='stripe_test_secret_key' value=settings.stripeTestSecretKey}}
       </div>
       <div class="field">
         <label>
           {{t 'Publishable Test Key'}}
+          {{ui-popup tagName="i" class="info icon" content="This publishable key will be used for testing mode"}}
```
Comment (CosmicCoder96) [inline app/templates/components/forms/admin/settings/payment-gateway-form.hbs]: translation helpers

File: app/templates/components/forms/admin/settings/payment-gateway-form.hbs [hunk 3]
```diff
@@ -55,15 +58,24 @@
                  onChange=(action (mut stripeMode))}}
     </div>
     {{#if (eq stripeMode 'production')}}
+      <div class="field">
+        <label>
+          {{t 'Client ID'}}
+          {{ui-popup tagName="i" class="info icon" content="This client id will be used during live (production) mode"}}
```
Comment (CosmicCoder96) [inline app/templates/components/forms/admin/settings/payment-gateway-form.hbs]: translation helpers

File: app/templates/components/forms/admin/settings/payment-gateway-form.hbs [hunk 4]
```diff
@@ -55,15 +58,24 @@
...
+        </label>
+        {{input type='text' name='stripe_client_id' value=settings.stripeClientId}}
+      </div>
       <div class="field">
         <label>
           {{t 'Secret Key'}}
+          {{ui-popup tagName="i" class="info icon" content="This Secret Key will be used during live (production) mode"}}
```
Comment (CosmicCoder96) [inline app/templates/components/forms/admin/settings/payment-gateway-form.hbs]: translation helpers

File: app/templates/components/forms/admin/settings/payment-gateway-form.hbs [hunk 5]
```diff
@@ -55,15 +58,24 @@
...
         </label>
         {{input type='text' name='stripe_secret_key' value=settings.stripeSecretKey}}
       </div>
       <div class="field">
         <label>
           {{t 'Publishable Key'}}
+          {{ui-popup tagName="i" class="info icon" content="This Publishable Key will be used during live (production) mode"}}
```
Comment (CosmicCoder96) [inline app/templates/components/forms/admin/settings/payment-gateway-form.hbs]: translation helpers

**Comment by kushthedude** [comment]
@CosmicCoder96 Done

**Comment by kushthedude** [comment]
@uds5501 @shreyanshdwivedi @mrsaicharan1 Please Review

**Pull request merged**